projects/plugins/jetpack/extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -141,6 +141,21 @@ function process_anchor_params() {
 							),
 						);
 					}
+				} else {
```

Comment:
Should we also check for `$valid_spotify_url` here, to avoid doing anything when the URL is not valid?